examples/hybrid/App.tsx

```diff
@@ -23,13 +26,47 @@ const throwUnhandled = () => {
   throw Error('This is an unhandled JS Crash');
 };
 
+const sendGraphQLRequest = async () => {
+  try {
+    const response = await axios.post(
+      'https://countries.trevorblades.com/graphql',
+      {
+        query: `
+        query GetCountry {
+          country(code: "EG") {
+            emoji
+            name
+          }
+        }
+      `,
+      },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          'ibg-graphql-header': 'GetCountry',
+        },
+      },
+    );
+    console.log('Response:', response.data.data);
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
 const App: () => ReactNode = () => {
   useEffect(() => {
     Instabug.init({
-      token: 'deb1910a7342814af4e4c9210c786f35',
+      token: '0fcc87b8bf731164828cc411eccc802a',
```

Comment:
Which project is this token for?

Reply:
React Native beta app https://dashboard.instabug.com/applications/reactnative/beta/settings/sdk-integrations